src/app/modules/main-module/components/main/main.component.ts

```diff
@@ -76,12 +87,18 @@ export class MainComponent implements OnInit {
   }
 
   writeLetter(tecla: string) {
-    this.word[this.positionInput] = tecla;
-    this.positionInput = this.findCorrectIndex();
+    this.word[this.rounds[this.contRound].positionInput] = tecla;
```

Comment:
el utilizar una variable y asignarle this.rounds[this.contRound].positionInput va a hacer que sea mas legible las lineas y resumir que significa ese chorizaco